packages/clay-color-picker/src/index.tsx

```diff
@@ -281,43 +286,30 @@ const ClayColorPicker: React.FunctionComponent<IProps> = ({
 								])}
 								disabled={disabled}
 								insetBefore
-								onBlur={(event) => {
-									const newColor = tinycolor(
-										event.target.value
-									);
-
-									const hexString = newColor.isValid()
-										? newColor.toHex()
-										: event.target.value;
-
-									onValueChange(hexString);
-									setHexInputValue(hexString);
-									onBlur(event);
-								}}
 								onChange={(event) => {
-									const newHexValue = event.target.value;
-
-									const newColor = tinycolor(newHexValue);
-
-									setHexInputValue(newHexValue);
-
-									if (newColor.isValid()) {
-										onValueChange(newColor.toHex());
-									}
+									onValueChange(event.target.value);
 								}}
 								ref={inputRef}
 								type="text"
-								value={hexInputValue
-									.toUpperCase()
-									.substring(0, 6)}
+								value={value}
 							/>
 
 							<ClayInput.GroupInsetItem before tag="label">
-								{'#'}
+								{isHex ? (
+									'#'
+								) : (
+									<ClayIcon
+										spritemap={spritemap}
+										style={{width: 10}}
+										symbol="color-picker"
+									/>
+								)}
 							</ClayInput.GroupInsetItem>
 						</ClayInput.GroupItem>
 					)}
 				</ClayInput.Group>
+
+				{feedbackContent}
```

Comment:
@bryceosterhaus to prevent us from adding this new API maybe the dev can compose with `ClayForm.Group` to create the validation on the outside, this can work better and give the more control.

```jsx
<ClayForm.Group className="has-error">
	<ClayColorPickerWithState
		{...props}
		colors={customColors}
		onColorsChange={setCustoms}
		onValueChange={setColor}
		value={color}
	/>
	<ClayForm.FeedbackGroup>
		<ClayForm.FeedbackItem>
			{"This is a form-feedback-item."}
		</ClayForm.FeedbackItem>
		<ClayForm.FeedbackItem>
			<ClayForm.FeedbackIndicator
				spritemap={spritemap}
				symbol="exclamation-full"
			/>
			{"This is a form-feedback-item with a error feedback indicator."}
		</ClayForm.FeedbackItem>
	</ClayForm.FeedbackGroup>
</ClayForm.Group>
```

Reply:
When I looked at it I was even thinking that the `title` API could also be avoided 🤔

Reply:
Since we already provide the title API, I'm just gonna leave that as is, but I like the idea with the feedback. I'm gonna create a storybook example for that as well.

Reply:
Yeah, maybe we can plan to remove the `title` API in an upcoming major version.